src/facade/facade.cc

```diff
@@ -137,12 +137,6 @@ ConnectionContext::ConnectionContext(::io::Sink* stream, Connection* owner) : ow
   subscriptions = 0;
 }
 
-RedisReplyBuilder* ConnectionContext::operator->() {
-  CHECK(Protocol::REDIS == protocol());
```

Comment:
Now this check was removed , but I think its an important one.
Can we instead of static casting in every place have one function for casting? which will not be the operator () and in this funciotn we will have this check?

Reply:
I don't see any point in this check, because we do casting on the commands that exist only in Redis.
I have thought about providing 2 methods asRedisReplyBuilder() and asMCReplybuilder(), but then have decided that such methods spoil the connectionContext interface